code/sorting/src/cocktail_sort/cocktail_sort.py

```diff
@@ -0,0 +1,46 @@
+# Python program for implementation of Cocktail Sort 
+# Cocktail sort traverses via both the directions alternatively
+# It works similar to bubble sort. But takes less iterations/pass.
+
+def cocktailSort(a):
+    n = len(a)
+    # swapped flag is set to True to enter the while loop
+    swapped = True
+    # starting index of the list
+    start = 0
+    # last index in the list
+    end = n - 1
```

Comment:
The comments on lines 9, and 11 are unnecessary. The comment on line 7 should explain the purpose of the variable, not what its set to.